src/bci_build/package.py

```diff
@@ -2230,6 +2230,39 @@ def _create_php_bci(
     for os_version in ALL_BASE_OS_VERSIONS
 ]
 
+GIT_CONTAINERS = [
+    ApplicationStackContainer(
+        name="git",
+        os_version=os_version,
+        support_level=SupportLevel.L3,
+        package_name="git-image",
+        pretty_name=f"{os_version.pretty_os_version_no_dash} with Git",
+        custom_description="A micro environment with Git for containers based on the SLE Base Container Image.",
+        from_image=f"{_build_tag_prefix(os_version)}/bci-micro:{OsContainer.version_to_container_os_version(os_version)}",
```

Comment:
Just out of curiosity, why not busybox?

Reply:
iirc it didn't work. I can try again, don't recall the error message